cvat-sdk/cvat_sdk/core/uploading.py

```diff
@@ -273,17 +279,19 @@ def upload_file_and_wait(
     ):
         url = self._client.api_map.make_endpoint_url(endpoint.path, kwsub=url_params)
         params = {"format": format_name, "filename": filename.name}
-        self.upload_file(
+        response = self.upload_file(
             url, filename, pbar=pbar, query_params=params, meta={"filename": params["filename"]}
         )
 
+        assert json.loads(response.data).get("rq_id"), "The rq_id was not found in the response"
```

Comment:
I don't understand this. Shouldn't the `rq_id` be used somehow? Why would we check that it's there and then not use it?